data/korath/korath weapons.txt

```diff
@@ -53,6 +54,64 @@ effect "grab-strike impact"
 	"velocity scale" -.1
 
 
+outfit "Korath Shunt-Strike"
+	category "Turrets"
+	cost 821000
+	thumbnail "outfit/shunt-strike"
+	"mass" 42
+	"outfit space" -42
+	"weapon capacity" -42
+	"energy capacity" 260
+	"turret mounts" -1
+	"required crew" 1
+	"asteroid scan power" 16
+	weapon
+		sprite "projectile/shunt-strike"
+			"frame rate" 24
+			"random start frame"
+		"hardpoint sprite" "hardpoint/shunt-strike"
+		"hardpoint offset" 7.
+		sound "shunt-strike"
+		"inaccuracy" 1
+		"turret turn" 2.1
+		"velocity" 10
+		"lifetime" 81
+		"reload" 30
+		"hit effect" "shunt-strike hit"
+		"die effect" "shunt-strike sparkle"
+		"firing energy" 100
+		"firing heat" 150
+		"relative firing shields" 0.001
+		"acceleration" 1
+		"drag" .1
+		"turn" .2
+		"homing" 4
+		"tracking" .9
+		"shield damage" 48
+		"corrosion damage" 3
+		"minable damage" 200
+		"piercing" 0.1
+		"blast radius" 30
+		"safe"
+		"missile strength" 80
+	description "This weapon fires homing balls of microbots that pass through a target's shields and attempt to convert its hull into raw materials or useful items such as heat shunts. They inevitably fail, as Exile microbots cannot sustain themselves without a Systems Core, but not before wreaking havoc on the ship."
+	description "Because the same property that allows the microbots through a target's shields disrupts the shields of the ship firing them, it is never used without an appropriately large Systems Core. Given adequate shield generation, however, the microbots make fantastic miners."
+
+effect "shunt-strike hit"
+	sprite "effect/shunt-strike-hit"
+		"no repeat"
+		"frame rate" 12
+	"lifetime" 30
+	"velocity scale" 0.05
+
+effect "shunt-strike sparkle"
+	sprite "effect/shunt-strike-sparkle"
+		"frame rate" 30
+	"lifetime" 30
+	"velocity scale" 0.05
+	"random angle" 10
+
+
```

Comment:
Whilst this is quite comical to picture a target ship being ripped apart and essentially doing a chunk of the korath's manufacturing in space, I'm not entirely sure about the name. Personally I'd suggest changing the name of this weapon to something like `"System Destruction Swarm"`? (please feel free to use a better name than this though)

Reply:
Well, the idea is that this is also a "-strike" weapon, in line with the Grab-Strike (I think?) and System Destruction Swarm sounds too much like a handful.

Reply:
I'd rather not make X-strike a korath weapon series. Grab-Strike on its own is an awkward name, I'd rather rename both weapons. ...Not that I have replacement names.

Reply:
I think awkward is good, it shows these names have been translated out of a very different tounge.  It's probably a single word made out of the two in Korath, like you often see in German, which I understand played a role in its inspiration.

However I do agree with Derpy in that the Korath- prefix needs to go when it comes to item names.

Reply:
Yeah I'll back the language-origin justification. Additionally, since this invariably suggests that the ship is capable of producing these microbot balls somewhat reliably I note that this supports my suggestion of explicitly making the systems core descriptions reference *producing* the bots, not just housing them.

Reply:
> I think awkward is good, it shows these names have been translated out of a very different tounge. It's probably a single word made out of the two in Korath, like you often see in German, which I understand played a role in its inspiration.
> 
> I don't speak any german at all, but I notice when you put two words like this next to each other in google translate, it comes out to a single, short word in German if you use a hyphen, and a longer single word if you don't.
> 
> However I do agree with Derpy in that the Korath- prefix needs to go when it comes to item names.

Ok, well it looks like shunt strike is a deliberate choice your end. I'm not attached to it so feel free to leave it as such.